launch/launch/frontend/entity.py

```diff
@@ -116,3 +116,8 @@ def assert_entity_completely_parsed(self):
         function completed.
         """
         raise NotImplementedError()
+
+    @property
+    def bare_text(self) -> Optional[Text]:
+        """Return the bare text of this element if it is a bare text element, or None."""
+        raise NotImplementedError()
```

Comment:
Can we find a more descriptive name for this, or just describe it using words other than "bare text" in the docstring, so that it's a bit clearer?